style.css

```
@@ -0,0 +1,158 @@
+* {
+  box-sizing: border-box;
+}
+
+:root{
+  --maincolor: #cfe5ff;
+  --maintextcolor_blue: #3692ff;
+  --footlinkcolor : #e5e7eb;
+  --footcopyright : #9ca3af;
+}
+
+span{
+  color: var(--maintextcolor_blue);
```

스네이크케이스라면 이렇게 쓰는편이 좋겠습니다
```suggestion
  color: var(--main_textcolor_blue);
```